nplbam/app/templates/dashboard.html

```diff
@@ -2,6 +2,9 @@
 {% block css %}animals{% endblock %}
 {% block main %}
 <div>
+    <div id="button">
+        <a href="/visualize">Visualize</a>
+    </div>
```

Comment:
Added here since this is an admin only page and didn't want to clog the nav too much